src/main/java/org/synyx/urlaubsverwaltung/security/SecurityWebConfiguration.java

```diff
@@ -48,9 +50,33 @@ SecurityFilterChain actuatorSecurityFilterChain(final HttpSecurity http) throws
     }
 
     @Bean
-    SecurityFilterChain webSecurityFilterChain(final HttpSecurity http, DelegatingSecurityContextRepository securityContextRepository) throws Exception {
+    @Order(3)
```

Comment:
Why do we need higher priority here? URL patterns do not match to others in general securityFilterChain.

Reply:
That we have a fixed order. Makes it more reliable on changes